app/src/components/BCIDView.tsx

```diff
@@ -262,12 +72,18 @@ const BCIDView: React.FC = () => {
     outputRange: ['0deg', '360deg'],
   })
 
+
+  const onGetIdTouched = () => {
+    setWorkflowInFlight(true)
+    startFlow(agent, store, dispatch as React.Dispatch<ReducerAction<any>>, setWorkflowInFlight, t, setAgentDetails)
```

Comment:
Characters are free, its helpful to create self documenting code by using more verbose names like `startGetBCPersonCredentialWokflow` <- just an example.